services/site/src/pages/index.tsx

```diff
@@ -80,67 +73,123 @@ const Home: React.VoidFunctionComponent<HomeProps> = ({ fmType }) => {
-        focus-rounded-instead-of-underline"
-              >
-                Twitter account
-              </InTextLink>
-              . <br />
-            </p>
-
-            {dataChallengesInProgress?.challenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={"Continue where you left"}
-                challenges={dataChallengesInProgress.challenges}
-                displayCompleted={false}
-              />
+      <Navigation />
+      <main className="bg-texture bg-repeat-y bg-contain bg-origin-border bg-top">
+        <div className={clsx("h-full max-w-screen-3xl px-8", "sm:px-12", "md:px-24", "xl:px-24", "2xl:mx-auto")}>
+          <HeroSection>
+            <IllustrationFloatingWoman className={clsx("h-auto move-floating-woman", animation && "stopAnimation")} />
+            <Button
+              srText={animation ? "Start animation" : "Stop animation"}
+              overrideClassName
+              animation
+              className={clsx("md:mb-20")}
+              onClick={() => setAnimation((prevAnimation) => !prevAnimation)}
+            >
+              {animation ? "Start" : "Stop"}
+            </Button>
+          </HeroSection>
+          <MediaSection />
+          <TopChallengeSection>
+            {topChallenges.data && (
+              <>
+                <TopChallenge challenge={topChallenges.data.challenge1} timesCompleted="3400" isUserFavorite />
+                <TopChallenge challenge={topChallenges.data.challenge2} timesCompleted="1700" />
+                <TopChallenge challenge={topChallenges.data.challenge3} timesCompleted="60" isNew />
+              </>
             )}
-
-            {easyChallenges.length > 0 && (
-              <ChallengeList
-                className={clsx("2xl:mx-24")}
-                heading={dataChallengesInProgress?.challenges.length === 0 ? "All challenges" : "Other challenges"}
-                challenges={easyChallenges}
+          </TopChallengeSection>
+          <IconSection />
+          {!currentUser?.isRegistered && <SignUpSection />}
+          <div className="my-8">
+            <USPSection
+              imageLeft
+              heading="Study from the comfort of your home"
+              paragraph="For challenges on a11yphant you won't need to read large amounts. Instead, you will learn by applying the concepts in code."
+            >
+              <div className="flex flex-row-reverse md:flex-row justify-start min-w-[46%] md:max-w-xs lg:max-w-sm xl:max-w-full lg:justify-end lg:pr-12">
+                <Button
+                  srText={animation ? "Start animation" : "Stop animation"}
+                  overrideClassName
+                  animation
+                  className={clsx("mb-52 xs:ml-6 md:ml-0 md:mb-0")}
+                  onClick={() => setAnimation((prevAnimation) => !prevAnimation)}
+                >
+                  {animation ? "Start" : "Stop"}
+                </Button>
```

Comment:
Could be moved to a component for easier adjustments in the future. But you don't have to